### DISCUSSION TITLE
Are Case Rates Algorithms?

### DISCUSSION THREAD

**Discussion by vduffel99** [CMSgov/hospital-price-transparency#95] [Q&A] [answered]
I am trying to determine if case rates (DRGs, Groupers, APC, ASC, etc) should be considered an algorithm. 

Per the [88 FR 82096](https://www.federalregister.gov/d/2023-24293/p-4970) “If the hospital has established a standard charge as a percentage or algorithm such that a standard dollar amount is not available, then the hospital would display the standard charge as a percentage or algorithm”. In this context, DRGs, Groupers, APC, ASC, etc. have a standard charge. There is a payer base rate and a weight and the standard dollar amount is available as it can be calculated the same for all patients. Using this logic I would comprehend the statement that DRGs, Groupers, APC, ASC, etc. would not be considered an algorithms.

The section goes on to explain that “An example of where we would expect to see a standard charge expressed as an algorithm would be when a hospital has negotiated reimbursement for defined service packages (for example, hip replacement or colonoscopy) that are based on differential percentages of total billed (gross) charges (for example, 50 percent of total billed charges for hip replacement and 75 percent of total billed charges for colonoscopy)” In this context, this does not describe the base rate * weight scenario (which would be the same for every patient). 

In the final action section, the last sentence states “Descriptions for algorithms could include, for example, a link to the algorithm used, a descriptor of a commonly understood algorithm or a list of factors that would be used to determining the individualized or variable allowed amount in dollars”. In this context, thoughts would be around the individualized or variable amount. If the standard charge negotiated dollar is not individualized or a variable, it would not be considered an algorithm.

So the question would really be if a DRGs, Groupers, APC, ASC, etc. would expect the same reimbursement (by service) for all patients, would it be considered an algorithm?

**Comment by RidenourK** [comment, marked as answer]
@vduffel99. Thank you for your question.

As indicated in the CY 2024 OPPS/ASC Final Rule, based on our experience in enforcing the requirements of the regulation, we have learned that most commercial contracting methods result in a hospital's ability to identify and display as a dollar figure the payer-specific negotiated charges they have established with third party payers. For example, a negotiated rate is established as a dollar amount for an item or service or service package (that is, the `base rate'), or is established as a percent discount off the gross charge for each item or service provided, or as a percentage of the Medicare rate which can be translated and displayed by the hospital as a standard dollar amount.

At other times, however, hospitals and payers establish the payer-specific negotiated charge by agreeing to an algorithm that will determine the dollar value of the allowed amount on a case-by-case basis after a pre-defined service package has been provided. This means that the standard charge that applies to the group of patients in a particular payer's plan can only prospectively be expressed as an algorithm, because the resulting allowed amount in dollars will be individualized on a case-by-case basis for a pre-defined service package, and thus cannot be known in advance or displayed as a rate that applies to each member of the group…. when no standard dollar amount is available, we have allowed hospitals to make public the standard algorithm that applies to the group. When a hospital has established a payer-specific negotiated charge that can only be expressed as a percentage or algorithm, it must display alongside that percentage or algorithm an ‘estimated allowed amount' in dollars for that payer/plan for that particular item or service. Please refer to the discussion beginning at [88 FR 82099](https://www.federalregister.gov/d/2023-24293/p-4979).

Please refer to the “Examples” available on this the CMS Hospital Price Transparency - Data Dictionary GitHub repository for examples of different ways hospitals could encode DRGs.

**Comment by vduffel99** [comment]
This information was helpful. I appreciate you taking the time to answer my questions

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
